response.go

```diff
@@ -586,6 +586,31 @@ func NewJsonResponse(status int, body any) (*http.Response, error) { // nolint:
 	return response, nil
 }
 
+// NewJsonResponseOrPanic is like [NewJsonResponse] but panics in case of error.
+//
+// It simplifies the call of [ResponderFromMultipleResponses], avoiding the
+// use of a temporary variable and an error check, and so can be used in such context:
```

Comment:
I didn't specified in the previous comment for the same line, but try to stay below 80 columns in comments:
```suggestion
// use of a temporary variable and an error check, and so can be used in such
// context:
```

Reply:
Will keep that in mind !